### PR TITLE
clone x times

### PR DESCRIPTION
**Pull request by davidackroyd99**
This works pretty nicely.

Notes:

- Might want to consider renaming FramesPrompt to something more generic like IntPrompt as I'm using it to get a number for something else
- Haven't done the keyboard shortcut because I don't know how, might do it tomorrow night if I get chance

### CODE REVIEW

**Comment by YoshiRulz** [comment]
```
git checkout feature/clone-x-times
git branch bak
git rebase -i master # assuming master is tracking TASVideos/BizHawk:master and you've not messed with it
# (delete the irrelevant commits)
git push -f origin feature/clone-x-times # assuming origin is your fork
```

File: src/BizHawk.Client.EmuHawk/tools/TAStudio/TAStudio.cs [hunk 1]
```diff
@@ -500,6 +500,7 @@ public void AddColumn(string columnName, string columnText, int columnWidth, Col
 		public void InsertNumFramesExternal() => InsertNumFramesMenuItem_Click(null, null);
 		public void DeleteFramesExternal() => DeleteFramesMenuItem_Click(null, null);
 		public void CloneFramesExternal() => CloneFramesMenuItem_Click(null, null);
+		public void CloneFramesXTimesExternal() => CloneFramesXTimesMenuItem_Click(null, null);
```
Comment (adelikat) [inline src/BizHawk.Client.EmuHawk/tools/TAStudio/TAStudio.cs:503]: This isn't necessary, it's only if we need to run this functionality elsewhere, such as lua

Comment (vadosnaprimer) [reply src/BizHawk.Client.EmuHawk/tools/TAStudio/TAStudio.cs:503]: Isn't it used for hotkeys?

Comment (adelikat) [reply src/BizHawk.Client.EmuHawk/tools/TAStudio/TAStudio.cs:503]: that too, but none of that is wired up in this PR, so I don't think this line should be added, until it is needed

File: src/BizHawk.Client.EmuHawk/tools/TAStudio/FramesPrompt.Designer.cs [hunk 2]
```diff
@@ -91,7 +91,7 @@ private void InitializeComponent()
 			this.Name = "FramesPrompt";
 			this.ShowIcon = false;
 			this.StartPosition = System.Windows.Forms.FormStartPosition.CenterParent;
-			this.Text = "Insert Frames";
+			this.Text = headMessage;
```
Comment (adelikat) [inline src/BizHawk.Client.EmuHawk/tools/TAStudio/FramesPrompt.Designer.cs:94]: Does this work in the designer?

Comment (YoshiRulz) [reply src/BizHawk.Client.EmuHawk/tools/TAStudio/FramesPrompt.Designer.cs:94]: It won't, this assignment needs to be the empty string and another assignment can be done in the ctor after `InitializeComponent`.

Comment (davidackroyd99) [reply src/BizHawk.Client.EmuHawk/tools/TAStudio/FramesPrompt.Designer.cs]: I've resolved this, thanks for the input

**Pull request merged**